main.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"database/sql"
+	"fmt"
+
+	"github.com/gin-gonic/gin"
+	_ "github.com/go-sql-driver/mysql"
+	"golang.org/x/crypto/bcrypt"
+)
+
+var db *sql.DB
+
+func main() {
```

Comment:
Create a Makefile for this project
Check on chat gpt what a makefile is and then ask it to create some basic commands for building your code and running your code. 
And check usage as well

Reply:
Read about makefile, looks interesting